File: tournaments/onlyEvenNumbers/onlyEvenNumbers.cs
int[] onlyEvenNumbers(int left, int right) {
    var result = new List<int>();
    
    for (int i = left; right >= i; ++i) {
        if (0 == (i & 1)) {
            result.Add(i);
        }
    }
    
    return result.ToArray();
}

File: tournaments/onlyEvenNumbers/onlyEvenNumbers.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub onlyEvenNumbers {
    my ($left, $right) = @_;
    my @result;

    for (my $i = $left; $right >= $i; ++$i) {
        unless ($i & 1) {
            push(@result, $i);
        }
    }

    return \@result;
}
